frontend/src/app/modules/common/no-results/no-results.component.ts

```diff
@@ -26,7 +26,8 @@
 // See docs/COPYRIGHT.rdoc for more details.
 //++
 
-import {Component, Input, HostBinding} from '@angular/core';
+import {Component, HostBinding, Input} from '@angular/core';
+
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/c30cd351cbaae9715a5df0fcb89c701d4b71b4d0/frontend/src/app/modules/common/no-results/no-results.component.ts#L31"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>